core/benches/shredder.rs

```diff
@@ -27,14 +28,18 @@ fn make_large_unchained_entries(txs_per_entry: u64, num_entries: u64) -> Vec<Ent
         .map(|_| make_test_entry(txs_per_entry))
         .collect()
 }
+const SHRED_SIZE_TYPICAL: usize = {
```

Comment:
Maybe `DEFAULT_SHRED_SIZE` instead?

Reply:
Its not "default". There is no default size unfortunately. Typical = most probable. Real size varies with configurations.